src/Intacct/Functions/AccountsPayable/AbstractVendor.php

```diff
@@ -549,7 +572,7 @@ public function getPrimaryEmailAddress()
      */
     public function setPrimaryEmailAddress($primaryEmailAddress)
     {
-        if (filter_var($primaryEmailAddress, FILTER_VALIDATE_EMAIL) === false) {
+        if (!empty($primaryEmailAddress) && filter_var($primaryEmailAddress, FILTER_VALIDATE_EMAIL) === false) {
```

Comment:
do u need `!empty`?

Reply:
I looked into other objects where we're doing something similar and this seems to be the approach of how we were allowing the unsetting of an already set value in the SDK